lib/takeaway.rb

```diff
@@ -0,0 +1,48 @@
+require 'twilio-ruby'
+
+class Takeaway
+  attr_reader :order, :total
+
+  def initialize
+    @menu = {
+      "fish and chips" => 3.20, 
+      "burger and chips" => 4.20, 
+      "chips" => 1
+    }
+    @order = {}
+    @total = 0
+  end
+  
+  def menu
+    @menu.each { |k, v| puts "#{k} - £%.2f" % v }
+  end
+
+  def add_meal(meal_selection, quantity)
```

Comment:
This method is quite hard to understand.  It would be worth focussing on how to improve readability here and, also, that might make you re-examine some of your design choices.